ecs-cli/modules/cli/configure/configure.go

```diff
@@ -44,7 +109,10 @@ func Cluster(context *cli.Context) error {
 		return err
 	}
 
-	clusterConfig := &config.Cluster{Cluster: cluster, Region: region}
+	cfnStackName := context.String(command.CFNStackNameFlag)
+	composeServiceNamePrefix := context.String(command.ComposeServiceNamePrefixFlag)
```

Comment:
If cfnStackName is the default (amazon-ecs-cli-setup..), don't need to set this.
If composeServiceNamePrefix is empty, don't need to set this.

Reply:
Let's talk about this in person. I have some questions.

Reply:
For `compose-service-name-prefix`, there is no different between setting it and not setting it in the case where it is empty. That's because in Golang, all fields in a struct are set to their default value (which is "" for strings). 

For `cfn-stack-name`, I'm updating that now to set it only if it isn't the default value.